### PR TITLE
Login & Registration

### PR DESCRIPTION
**Pull request by avrey-s**
Includes functions to register a new user, as well as login, applying Zeeshs previous styles to the stack display.

### CODE REVIEW

File: App.tsx
Comment (SwyftPain) [inline App.tsx]: This should go in the styles file

File: components/Others/styles.tsx
Comment (SwyftPain) [inline components/Others/styles.tsx]: More descriptive folder names should be used

File: components/Others/typeconfig.tsx
Comment (SwyftPain) [inline components/Others/typeconfig.tsx]: More descriptive folder names should be used

File: inputs/LoginForm.tsx
Comment (SwyftPain) [inline inputs/LoginForm.tsx]: Duplicate file, needs removal

File: inputs/RegistrationForm.tsx
Comment (SwyftPain) [inline inputs/RegistrationForm.tsx]: Duplicate file, needs removal

File: styles.tsx
Comment (SwyftPain) [inline styles.tsx]: Duplicate styles

**Review by SwyftPain — COMMENTED** [review]
This is prefferred to make changes rather than do a hotfix, however, we are way behind our deadline. 

I am unsure about AccountSettings, SalaryForm, TransactionForm and TransactionView in the inputs folder, but LoginForm and RegistationForm are duplicates. 

Styles is duplicate files both in the root and "Other directory and we have styling directly in the files. We should use styling from one of the styling files.

A folder called "Other" is bit non-descriptive.

I will wait on Zeesh's input before making the final decision :)

File: inputs/RegistrationForm.tsx
Comment (zsh28) [inline inputs/RegistrationForm.tsx:1]: as swyft mentioned, duplicate files need to be removed

**Review by zsh28 — CHANGES_REQUESTED** [review]
I believe following swyft recommendations will be beneficial rather than creating a hotfix for the duplicate files.

**Review by SwyftPain — CHANGES_REQUESTED** [review]
Alright, then based on Zeesh's decision, doing by the best practices method :)

File: App.tsx
Comment (avrey-s) [reply App.tsx]: Removed excess in most recent commit

File: components/Others/styles.tsx
Comment (avrey-s) [reply components/Others/styles.tsx]: Created 2 different folders and named them accordingly to resolve in most recent commit

File: inputs/LoginForm.tsx
Comment (avrey-s) [reply inputs/LoginForm.tsx]: Deleted duplicate file in most recent commit

File: components/Navigation/Home.tsx
Comment (SwyftPain) [inline components/Navigation/Home.tsx]: This could have been a simple default file just for the sake of pushing navigation (i.e. default App.tsx) and then doing this as separate PR, alas, for the sake of completing the project, it is fine.

File: package.json
Comment (SwyftPain) [inline package.json:21]: This should be it's on PR but lets just do it anyway, because its causing us problems

**Review by SwyftPain — APPROVED** [review]
So, a lot of extra files weren't really needed, could have been default files and just pushed a navigation itself and then worked on those screens/pages later. Alas, for the sake of completing the project, it is fine.

Also, there are too styles which may lead to confusion, one in Styles/styles.tsx and another just styles.tsx.

Pick one and stick with it. Don't need both. Its getting very confusing.
Hope one of you picks this up and deals with it in a hotfix next, so you dont split and write some of your styles in one file and some in other and then getting conflicts and problems.

**Review by zsh28 — APPROVED** [review]
The code seems fine for me no issues really.

**Pull request merged**
